admin/helpers/browserSecurity.js

```diff
@@ -6,7 +6,7 @@ const config = require('../config')
 const init = (app) => {
   app.use(helmet())
   app.use(helmet.noCache())
-  const scriptSources = ["'self'", "'unsafe-inline'", 'https://www.google-analytics.com', 'https://www.googletagmanager.com']
+  const scriptSources = ["'self'", "'unsafe-inline'", 'https://www.google-analytics.com', 'https://www.googletagmanager.com', 'https://az416426.vo.msecnd.net']
```

Comment:
hmm, i wonder if they might change this random looking URL 🤔 hope not 😄 could be tricky for us

Reply:
this looks indeed like it was generated and likely to change.